python/lar_constraints.py

```diff
@@ -1149,3 +1150,119 @@ def v705_const(self, row):
 			row["aus_1"] = "6"
 			row["aus_result_1"] = "17"
 		return row
+	
+	def v709_const(self, row):
+		"""1) If Street Address, City, and Zip Code is reported Exempt, then all three
+			 must be reported Exempt."""
+		if row["street_address"] == "Exempt" and row["city"] == "Exempt" and row["zip_code"] == "Exempt":
+			row["street_address"] = "Exempt"
+			row["city"] = "Exempt"
+			row["zip_code"] = "Exempt"
+		return row
+	
+	def v710_const(self, row):
+		"""1) Credit Score of Applicant or Borrower, Credit
+			Score of Co-Applicant or Co-Borrower, Applicant or
+			Borrower, Name and Version of Credit Scoring
+			Model, and Co-Applicant or Co-Borrower, Name and
+			Version of Credit Scoring Model must be reported
+			1111; and
+		   2) Applicant or Borrower, Name and Version of
+			Credit Scoring Model: Conditional Free Form Text
+			Field for Code 8 and Co-Applicant or Co-Borrower,
+			Name and Version of Credit Scoring Model:
+			Conditional Free Form Text Field for Code 8 must be
+			left blank."""
+		if row["app_credit_score"]=="1111":
+			row["co_app_credit_score"] = "1111"
+			row["app_score_name"] = "1111"
+			row["co_app_score_name"] = "1111"
+			row["app_score_name"] = ""
+			row["app_score_code_8"] = ""
+			row["co_app_score_name"] = ""
+			row["co_app_score_code_8"] = ""
+		return row
+
+
+	def v711_const(self, row):
+		"""1) If the Reason for Denial exemption election is
+			taken, Reason for Denial: 1 must be reported 1111;
+			and
+		   2) Reason for Denial: 2, Reason for Denial: 3,
+			Reason for Denial: 4, and Reason for Denial:
+			Conditional Free Form Text Field for Code 9 must be
+			left blank."""
+		if row["denial_1"]=="1111":
+			row["denial_1"]= "1111"
+			row["denial_1"]= "1111" 
+			row["denial_2"]= ""
+			row["denial_3"]= ""
+			row["denial_4"]= ""
+			row["denial_code_9"]= ""
+		return row
+
+
+	def v712_const(self, row):
+		"""1) If the Total Loan Costs or Total Points and Fees
+			  exemption election is taken, Total Loan Costs and
+			  Total Points and Fees must be reported Exempt."""
+		if (row["loan_costs"]=="Exempt" or row["points_fees"]=="Exempt"):
+			row["loan_costs"]= "Exempt"
+			row["points_fees"]= "Exempt"
+
+		return row
+
+	def v713_const(self, row):
+		"""1) Automated Underwriting System: 1 and
```

Comment:
For clarity, consider adding: If the Automated Underwriting System exemption
election is taken,